.docker/images/nginx/tests/src/FilesCacheTest.php

```diff
@@ -4,46 +4,43 @@
 
 use PHPUnit\Framework\TestCase;
 
-class FilesCacheTest extends TestCase
-{
+class FilesCacheTest extends TestCase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp(): void {
+    `docker-compose exec nginx mkdir -p /app/sites/default/files`;
+    foreach ($this->providerExpiredHeaderPath() as $parts) {
+      list($file, $path) = $parts;
+      `docker cp $path/$file $(docker-compose ps -q nginx):/app/web/sites/default/files/`;
+    }
+  }
 
   /**
    * Return a list of files to test.
    *
    * @return array
    *   File list.
    */
-  public function filesProvider()
-  {
+  public function providerExpiredHeaderPath() {
     $path = dirname(__DIR__);
     return [
-      ["autotest.jpg", "$path/resources/", 'max-age=2628001'],
```

Comment:
here and below - using single quotes `'` everywhere for strings, except where string interpolation is required.